cpp/ql/src/semmle/code/cpp/Enum.qll

```diff
@@ -46,7 +52,17 @@ class Enum extends UserType, IntegralOrEnumType {
 }
 
 /**
- * A C++ enum that is directly enclosed by a function.
+ * A C++ enum that is directly enclosed by a function. For example, the type
+ * `MyLocalEnum` in:
+ * ```
+ * void myFunction()
+ * {
+ *   enum MyLocalEnum
+ *   {
```

Comment:
Brace placement varies over this file. I prefer the style we settled on for `Class.qll`, so this example would become

```
void myFunction() {
  enum MyLocalEnum {
     MyLocalEnumConstant
  };
}
```

Reply:
Fixed.